src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -130,8 +132,10 @@ def _crawl(self) -> list[Table]:
                 tasks.append(partial(self._describe, catalog, database, table))
         catalog_tables, errors = Threads.gather(f"listing tables in {catalog}", tasks)
         if len(errors) > 0:
-            # TODO: https://github.com/databrickslabs/ucx/issues/406
-            logger.error(f"Detected {len(errors)} while scanning tables in {catalog}")
+            for _e in errors:
+                failure = ObjectFailure("tables", "table", "table_name")
```

Comment:
We have table name on the level of individual task

Reply:
See new version